yt_dlp/extractor/youtube.py

```diff
@@ -3782,7 +3789,9 @@ def _invalid_player_response(self, pr, video_id):
     def _extract_player_responses(self, clients, video_id, webpage, master_ytcfg, smuggled_data):
         initial_pr = ignore_initial_response = None
         if webpage:
-            if 'web' in clients:
+            if self.po_token:
+                ignore_initial_response = True
```

Comment:
```suggestion
```

The initial response may still work? If it's for a session generated from a browser user should be passing cookies, so the visitor ID will match token that way.

Reply:
Tested and this works